precommit/src/main/shell/personality/hadoop.sh

```diff
@@ -404,11 +404,34 @@ function personality_modules
   fi
 
   extra="-Ptest-patch ${extra}"
-
-  for module in $(hadoop_order ${ordering}); do
-    # shellcheck disable=SC2086
-    personality_enqueue_module ${module} ${extra}
+  OZONE_CHANGED=false
+  CORE_HADOOP_CHANGED=false
+  for module in $CHANGED_MODULES
```

Comment:
CHANGED_MODULES is an array, not a string.  You want "${CHANGED_MODULES[@]}" here. (Also, looks like a bug in shellcheck: it should have flagged that.)

Reply:
Thanks a lot, fixed. 


(BTW, It was not flagged, I don't know why: 

```
precommit/src/main/shell/personality/hadoop.sh:426:46: note: Double quote to prevent globbing and word splitting. [SC2086]
precommit/src/main/shell/personality/hadoop.sh:427:47: note: Double quote to prevent globbing and word splitting. [SC2086]
```

I tested with a simple shell script and similar structure was flagged there. I think the type of the CHANGED_MODULES is unknown here...)